lib/iris/etc/pp_save_rules.txt

```diff
@@ -761,14 +761,14 @@ THEN
 
 #MDI
 IF
-    cm.fill_value is not None
+    isinstance(cm.data, ma.core.MaskedArray)
```

Comment:
It looks like @pp-mo added a `cube.fill_value` in #2452 

So this change will undo the benefit we got from that PR